src/types/types.ts

```diff
@@ -578,6 +579,8 @@ export type EnqueuedTaskObject = {
 
 export type TaskObject = Omit<EnqueuedTaskObject, "taskUid"> & {
   uid: number;
+  /** The UID of the batch that the task belongs to (`null` for enqueued tasks) */
+  batchUid: number | null;
```

Comment:
Should we update the tests for this too?

Reply:
Added a test to check it's defined ✅